File: Week-02/Day-1/draw_chess_table.py
# Crate a program that draws a chess table like this
#
# % % % % 
#  % % % %
# % % % %
#  % % % %
# % % % %
#  % % % %
# % % % % 
#  % % % %
#
def chess_table():
    for lines in range(8):
        if lines %2 ==0:
            print("% % % %")
        else:
            print(" % % % %")

chess_table()

File: Week-07/Day-1/draw-chess-table.js
'use strict';

// Crate a program that draws a chess table like this
//
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % %
//  % % % %
// % % % % 
//  % % % %
//
let lineCount = 8
for (let i = 1; i <= lineCount; i++){
    console.log(' %'.repeat(lineCount/2));
    console.log(' ' + ' %'.repeat(lineCount/2));
}
